pkg/publish/layout_test.go

```diff
@@ -25,25 +25,52 @@ import (
 )
 
 func TestLayout(t *testing.T) {
-	img, err := random.Image(1024, 1)
-	if err != nil {
-		t.Fatalf("random.Image() = %v", err)
+	cases := []struct {
+		name string
+		tags []string
+		want error
+	}{
+		{
+			name: "no tags",
+			tags: nil,
+			want: nil,
```

Comment:
`want` is `nil` in all of these cases. If none of these cases test the error scenario we can just remove `want` from the struct here.